src/test/java/com/redislabs/redisgraph/RedisGraphAPITest.java

```diff
@@ -12,247 +13,77 @@ public RedisGraphAPITest() {
-        int DIR_OUT = 0;
-        int DIR_IN = 1;
-        int DIR_BOTH = 2;
-
-        List<RedisEdge> edges;
-        edges = api.getNodeEdges(roi.getId(), "knows", DIR_OUT);
-        Assert.assertEquals(edges.size(), 2);
-
-        edges = api.getNodeEdges(roi.getId(), "knows", DIR_IN);
-        Assert.assertEquals(edges.size(), 2);
-
-        edges = api.getNodeEdges(roi.getId(), "knows", DIR_BOTH);
-        Assert.assertEquals(edges.size(), 4);
-
-        edges = api.getNodeEdges(amit.getId(), "knows", DIR_OUT);
-        Assert.assertEquals(edges.size(), 1);
-
-        edges = api.getNodeEdges(amit.getId(), "knows", DIR_IN);
-        Assert.assertEquals(edges.size(), 1);
-
-        edges = api.getNodeEdges(amit.getId(), "knows", DIR_BOTH);
-        Assert.assertEquals(edges.size(), 2);
-    }
-
-    @org.testng.annotations.Test
-    public void testGetNeighbours() throws Exception {
-        // Create both source and destination nodes
-        RedisNode roi = api.createNode("name", "roi", "age", 32);
-        RedisNode amit = api.createNode("name", "amit", "age", 30);
-        RedisNode shany = api.createNode("name", "shany", "age", 23);
-
-        // Connect source and destination nodes.
-        api.connectNodes(roi, "knows", amit);
-        api.connectNodes(roi, "knows", shany);
-        api.connectNodes(amit, "knows", roi);
-        api.connectNodes(shany, "knows", roi);
-
-        int DIR_OUT = 0;
-        int DIR_IN = 1;
-        int DIR_BOTH = 2;
-
-        List<RedisNode> neighbours;
-        neighbours = api.getNeighbours(roi.getId(), "knows", DIR_OUT);
-        Assert.assertEquals(neighbours.size(), 2);
-
-        neighbours = api.getNeighbours(roi.getId(), "knows", DIR_IN);
-        Assert.assertEquals(neighbours.size(), 2);
-
-        neighbours = api.getNeighbours(roi.getId(), "knows", DIR_BOTH);
-        Assert.assertEquals(neighbours.size(), 4);
-
-        neighbours = api.getNeighbours(amit.getId(), "knows", DIR_OUT);
-        Assert.assertEquals(neighbours.size(), 1);
-
-        neighbours = api.getNeighbours(amit.getId(), "knows", DIR_IN);
-        Assert.assertEquals(neighbours.size(), 1);
-
-        neighbours = api.getNeighbours(amit.getId(), "knows", DIR_BOTH);
-        Assert.assertEquals(neighbours.size(), 2);
+        ResultSet resultSet = api.query("MATCH (a:qhuman)-[knows]->(:qhuman) RETURN a");
+        
+    	Assert.assertTrue(resultSet.hasNext());
+    	Assert.assertNull(resultSet.getStatistics().getStringValue(Label.NODES_CREATED));
+    	Assert.assertNull(resultSet.getStatistics().getStringValue(Label.PROPERTIES_SET));
+    	Assert.assertNull(resultSet.getStatistics().getStringValue(Label.RELATIONSHIPS_CREATED));
+    	Assert.assertNull(resultSet.getStatistics().getStringValue(Label.RELATIONSHIPS_DELETED));
+    	Assert.assertNotNull(resultSet.getStatistics().getStringValue(Label.QUERY_INTERNAL_EXECUTION_TIME)); 
+
+    	Record record = resultSet.next();
+    	Assert.assertEquals( "roi", record.getString(1));
```

Comment:
Can't assume order, position 1 for name, position 0 for age,
unless you specify the order of return attributes.
Use getString("name") and getString("age")